app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,18 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] Kindly implement a post's likes functionality in a separate `likes_controller` file.